### PR TITLE
Article list/bl 94

### PR DESCRIPTION
**Pull request by vegarrsm**
Finished tests for article listview.

### CODE REVIEW

File: test/blocs/resource_bloc/resource_list_state_test.dart [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('ResourceListState', () {
+    group('ResourceListLoading', () {
+      test('toString returns correct value', () {
+        expect(Loading().toString(), 'Loading()');
+      });
+    });
+    // group('CourseListSuccess', () {
```
Comment (pmAllport) [inline test/blocs/resource_bloc/resource_list_state_test.dart:11]: Remove

Comment (pmAllport) [inline test/blocs/resource_bloc/resource_list_state_test.dart:11]: remove commented out test

File: test/views/course_list_view_test.dart [hunk 2]
```diff
@@ -66,6 +66,7 @@ void mainTest() {
         ),
       );
       await tester.pump();
+      debugPrint(mockedLangCourseList[0].title);
```
Comment (pmAllport) [inline test/views/course_list_view_test.dart:69]: Remove

**Review by Lotfi-AL — APPROVED** [review]
LGTM

**Pull request merged**